src/hooks/UserPreference.graphql

```diff
@@ -0,0 +1,15 @@
+query UserOption($key: String!) {
```

Comment:
This query and mutation doesn't need to load the `id` field anymore because I changed the cache to use the `key` to generate `Option`s' unique id.